lib/Doctrine/Common/Collections/Collection.php

```diff
@@ -292,5 +291,32 @@ public function indexOf($element);
      *
      * @psalm-return array<TKey,T>
      */
-    public function slice($offset, $length = null);
+    public function slice(int $offset, ?int $length = null) : array;
+
+    public function count() : int;
```

Comment:
I'm 👎 on copying core interfaces just to add type hints

Reply:
This was already in the PR when I took it over and frankly I didn't understand why it was there since those interfaces are already defined upstream in PHP. I will remove them.